src/display/font_loader.js

```diff
@@ -77,6 +77,22 @@ var FontLoader = {
     ));
   },
 
+  get isEvalSupported() {
+    var evalSupport;
```

Comment:
Nit: if this is initialized to `false`, you can skip the `else` branch at https://github.com/mozilla/pdf.js/pull/6141/files#diff-c0a7c347be94f54892dea2ead8b3df4cR90.